timbot.py

```diff
@@ -51,61 +51,67 @@ def run_timbot():
 
 	# look for a message in the chat that starts with '@timbot .....'
 	if 'messages' in history:
-		for message in  history['messages']:
-			if 'text' in message:
-				message = message['text']
+		for data in  history['messages']:
+			if 'text' in data and 'user' in data:
```

Comment:
```suggestion
			if 'text' in data and 'user' in data and data['user'] != timbot_user_id_striped:
```

Reply:
That works too but i think its more readable separated. You are combined key checks and data checks in one statement.

Reply:
I agree with Dan; too many logic checks in a single line is a bad idea